README.md

```diff
@@ -39,6 +39,67 @@ Provides features to aid usage and discovery of modules, providing:
 
 The site can be accessed at http://localhost:5000
 
+## Running with docker-compose
```

Comment:
The README is generated from a README.in, so these changes will need to be made there as well (the README.md is generated using `scripts/generate_readme.py`

Reply:
Noted. I'll everything over there